domain/src/test/java/org/oppia/domain/classify/rules/dragAndDropSortInput/DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTest.kt

```diff
@@ -96,7 +98,9 @@ class DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTe
   }
 
   @Test
+    /* ktlint-disable max-line-length */
```

Comment:
Here you can use a single line comment for disabling ktlint check of 100 char thing at the end of the line which you want to disable.
`// ktlint-disable max-line-length`
Check here [Ktlint wiki page](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide)

Here current issue `Unexpected indentation (4) (should be 2)` but using one line comment solve this too.